requirements.txt

```diff
@@ -1,4 +1,3 @@
-Crypto
```

Comment:
what happened here?

Reply:
Reverted this change - the missing module error i was getting for `Crypto` was when i was still trying to test in pycharm